packages/react-accordion/src/components/AccordionHeader/useAccordionHeader.ts

```diff
@@ -59,53 +44,30 @@ export const useAccordionHeader = (
   const id = useId('accordion-header-', props.id);
   const panel = useDescendants(accordionItemDescendantContext)[1] as AccordionItemDescendant | undefined;
   const innerRef = React.useRef<HTMLElement>(null);
-  const state = mergeProps(
-    {
-      ref: useMergedRefs(ref, innerRef),
-      size: 'medium' as AccordionHeaderSize,
-      inline: false,
-      expandIcon: {
-        as: DefaultExpandIcon,
-        'aria-hidden': true,
-      },
-      button: {
-        id,
-        onClick: onAccordionHeaderClick,
-        'aria-disabled': disabled,
-        'aria-controls': panel?.id,
-      },
-      as: 'div',
-      role: 'heading',
-      expandIconPosition: 'start',
-      context: {
-        disabled: false,
-        open: false,
-        size: 'medium',
-        expandIconPosition: 'start',
-      },
+  const uninitializedState = {
```

Comment:
Either I change to `uninitializedState` in #18819 or you change it to `initialState` 😅 but we should try to be consistent

Reply:
I'll change it then